plugins/monstro/Monstro.cpp

```diff
@@ -1465,29 +1465,12 @@ MonstroView::MonstroView( Instrument * _instrument,
 	m_opViewButton -> setActiveGraphic( PLUGIN_NAME::getIconPixmap( "opview_active" ) );
 	m_opViewButton -> setInactiveGraphic( PLUGIN_NAME::getIconPixmap( "opview_inactive" ) );
 	ToolTip::add( m_opViewButton, tr( "Operators view" ) );
-	m_opViewButton -> setWhatsThis( tr( "The Operators view contains all the operators. These include both audible "
-										"operators (oscillators) and inaudible operators, or modulators: "
-										"Low-frequency oscillators and Envelopes. \n\n"
-										"Knobs and other widgets in the Operators view have their own what's this -texts, "
-										"so you can get more specific help for them that way. " ) );
 
 	PixmapButton * m_matViewButton = new PixmapButton( this, NULL );
 	m_matViewButton -> move( 125,0 );
 	m_matViewButton -> setActiveGraphic( PLUGIN_NAME::getIconPixmap( "matview_active" ) );
 	m_matViewButton -> setInactiveGraphic( PLUGIN_NAME::getIconPixmap( "matview_inactive" ) );
 	ToolTip::add( m_matViewButton, tr( "Matrix view" ) );
-	m_matViewButton -> setWhatsThis( tr( "The Matrix view contains the modulation matrix. Here you can define "
-										"the modulation relationships between the various operators: Each "
-										"audible operator (oscillators 1-3) has 3-4 properties that can be "
-										"modulated by any of the modulators. Using more modulations consumes "
-										"more CPU power. \n\n"
-										"The view is divided to modulation targets, grouped by the target oscillator. "
-										"Available targets are volume, pitch, phase, pulse width and sub-osc ratio. "
-										"Note: some targets are specific to one oscillator only. \n\n"
-										"Each modulation target has 4 knobs, one for each modulator. By default "
-										"the knobs are at 0, which means no modulation. Turning a knob to 1 causes "
-										"that modulator to affect the modulation target as much as possible. Turning "
-										"it to -1 does the same, but the modulation is inversed. " ) );
```

Comment:
Monstro makes up some 10% of the lines and it's good instructions. It gives a hint to what it could look like if the WhatsThis function is given some tender loving care instead.

Reply:
The instruction here are definitely good, but they are a lot (139 lines) and in the source code (not a good place).
The best place for them is a dedicated article on Monstro in the wiki, which already exists and is in a pretty good state: https://lmms.io/wiki/index.php?title=Monstro